README.md

```diff
@@ -30,16 +30,40 @@ Run an example. Notice that the output, in csv format, is printed to the termina
 music_box -e Chapman
 ```
 
-You can also run your own configuration
+Output can be saved to a file in csv file when no --output-format is passed
 
 ```
-music_box -c my_config.json
+music_box -e Chapman -o output.csv
 ```
 
-Output can be saved to a file
+Output can be saved to a file as csv file when --output-format csv is passed
 
 ```
-music_box -e Chapman -o output.csv
+music_box --output-format csv -e Chapman -o output.csv
+```
+
+Output can be saved to a file as netcdf file when --output-format netcdf is passed
+
+```
+music_box --output-format netcdf -e Chapman -o output.nc
+```
+
+Output can be saved to a file in csv file to output.csv when no output path is given but --output-format is csv
```

Comment:
Would it be possible to add backticks around the options such as `--output-format`, so they appears clearer?
Output can be saved to a file in csv file to output.csv when no output path is given but `--output-format` is csv